content/user/template/listing_columns.md

```diff
@@ -0,0 +1,21 @@
+---
+title: Columnar Layout for Product Listing pages 
+description: Displaying products in columns 
+category: template
+weight: 10
+---
+
+Since 1.5.7, Zen Cart has had a configuration setting called [Columns Per Row](/user/admin_pages/configuration/configuration_productlisting/#columns_per_row).  This setting allows you to display multiple columns of products per row on the Product Listing page. 
```

Comment:
`has had` might read better as just `has`?

Also note that the only templates that have this built-in (as far as I know) are responsive_classic and the bootstrap4 template.  Some older templates for older ZC versions might have an older variant of the column-grid-layout mod incorporated, but the v157 implementation simplifies the code a lot, and reduces config settings.

Reply:
Will fix on next update.

Reply:
**Note:** Your template must also support this setting.  Both the Responsive Classic and Bootstrap templates support this setting.  Many older templates use a version of the Column Grid Layout plugin, which is deprecated by this setting.